robotiq_3_finger_gripper_driver/src/robotiq_3_finger_gripper_driver_node_main.ros2.cpp

```diff
@@ -0,0 +1,16 @@
+#include <memory>
+
+#include <rclcpp/rclcpp.hpp>
+
+#include "robotiq_3_finger_gripper_driver_node.ros2.hpp"
+
+int main(int argc, char * argv[])
```

Comment:
This should be `char** argv` to be consistent with existing code

Reply:
Done in https://github.com/ToyotaResearchInstitute/tri_hardware_drivers/pull/14/commits/349ad4728d2f4bd362c7a8971c81102748c14663.